elastic/kibana
# Hide create spaces button when limit is reached

Resolves #159028 
Resolves #159047

## Summary

Hide create spaces button when limit is reached. 

## Screenshot

![Spaces-Management-Disabled](https://github.com/elastic/kibana/assets/190132/587dc47b-0377-4f72-8faa-7e6652cdab96)

## Testing

1. Set the maximum number of allowed spaces to 1

```yml
xpack.spaces.maxSpaces: 1
```

2. Verify that the create spaces button is hidden and that a callout is displayed
